sdcm/nemesis.py

```diff
@@ -2668,9 +2668,14 @@ def disrupt_mgmt_repair_cli(self):
         mgr_cluster = self.cluster.get_cluster_manager()
         mgr_task = mgr_cluster.create_repair_task()
         task_final_status = mgr_task.wait_and_get_final_status(timeout=86400)  # timeout is 24 hours
-        assert task_final_status == TaskStatus.DONE,\
-            f'Task: {mgr_task.id} final status is: {str(mgr_task.status)}.\nTask progress string: ' \
-            f'{mgr_task.progress_string(parse_table_res=False, is_verify_errorless_result=True).stdout}'
+        try:
```

Comment:
why not replacing try-except with if-else?
like: `if task_final_status == TaskStatus.DONE:`
```
else: 
            if task_final_status not in (TaskStatus.ERROR, TaskStatus.ERROR_FINAL):
                mgr_task.stop()
            raise
```